### PR TITLE
Add Release Notes for iOS Release 1.71.125

### PR DESCRIPTION
**Pull request by Uni-verse**
Resolves https://github.com/brave/brave-browser/issues/42214

## Submitter Checklist:

- [x] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [x] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [x] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A
  - [ ] https://github.com/brave/brave-browser/wiki/Brave-Wallet

## Test Plan:
N/A

### CODE REVIEW

File: CHANGELOG_iOS.md [hunk 1]
```diff
@@ -1,5 +1,31 @@
 # Changelog
 
+## [1.71.125](https://github.com/brave/brave-browser/releases/tag/v1.71.125)
+
+### Web3
+
+ - Added support for resolving new UD TLDs. ([#40543](https://github.com/brave/brave-browser/issues/40543))
+ - Fixed creating send transaction when sending any non-compressed Solana NFT. ([#40424](https://github.com/brave/brave-browser/issues/40424))
```
Comment (srirambv) [inline CHANGELOG_iOS.md:8]: Maybe reword to `Fixed inability to send any non-compressed Solana NFT from Brave Wallet. `

**Review by srirambv — CHANGES_REQUESTED** [review]
Once updated its all good for Web3

File: CHANGELOG_iOS.md [hunk 2]
```diff
@@ -1,5 +1,31 @@
...
+
+### General
+
+ - Implemented local vector search for the selection of key text segments of page and video context. ([#36801](https://github.com/brave/brave-browser/issues/36801))
+ - Added new Playlist UI to be default. ([#40911](https://github.com/brave/brave-browser/issues/40911))
+ - Added support for running iOS app in compatibility mode on VisionOS. ([#40579](https://github.com/brave/brave-browser/issues/40579))
+ - Added “it_IT” and “es_AR” to list of supported locales for Brave News. ([#40996](https://github.com/brave/brave-browser/issues/40996))
```
Comment (kjozwiak) [inline CHANGELOG_iOS.md:15]: Some users might know what the above country codes mean. I would use the full names. Example:

```
 - Added Italy and Argentina support for Brave News. ([#40996](https://github.com/brave/brave-browser/issues/40996))
```

File: CHANGELOG_iOS.md [hunk 3]
```diff
@@ -1,5 +1,31 @@
...
+### Web3
+
+ - Added support for resolving new UD TLDs. ([#40543](https://github.com/brave/brave-browser/issues/40543))
+ - Fixed creating send transaction when sending any non-compressed Solana NFT. ([#40424](https://github.com/brave/brave-browser/issues/40424))
+
+### General
+
+ - Implemented local vector search for the selection of key text segments of page and video context. ([#36801](https://github.com/brave/brave-browser/issues/36801))
+ - Added new Playlist UI to be default. ([#40911](https://github.com/brave/brave-browser/issues/40911))
+ - Added support for running iOS app in compatibility mode on VisionOS. ([#40579](https://github.com/brave/brave-browser/issues/40579))
```
Comment (kjozwiak) [inline CHANGELOG_iOS.md:14]: Should actually mention Brave rather than mentioning `iOS app`. For example:

```
 - Added support for running Brave in compatibility mode on VisionOS. ([#40579](https://github.com/brave/brave-browser/issues/40579))
```

File: CHANGELOG_iOS.md [hunk 4]
```diff
@@ -1,5 +1,31 @@
...
+## [1.71.125](https://github.com/brave/brave-browser/releases/tag/v1.71.125)
+
+### Web3
+
+ - Added support for resolving new UD TLDs. ([#40543](https://github.com/brave/brave-browser/issues/40543))
+ - Fixed creating send transaction when sending any non-compressed Solana NFT. ([#40424](https://github.com/brave/brave-browser/issues/40424))
+
+### General
+
+ - Implemented local vector search for the selection of key text segments of page and video context. ([#36801](https://github.com/brave/brave-browser/issues/36801))
```
Comment (kjozwiak) [inline CHANGELOG_iOS.md:12]: Should add some context re: where we added it. We should mention Leo to give more context. Something as per the following:

```
  - Implemented local vector search for the selection of key text segments of page and video context in Brave Leo. ([#36801](https://github.com/brave/brave-browser/issues/36801))
```

OR

```
 - Implemented Brave Leo local vector search for the selection of key text segments of page and video context. ([#36801](https://github.com/brave/brave-browser/issues/36801))
```

File: CHANGELOG_iOS.md [hunk 5]
```diff
@@ -1,5 +1,31 @@
...
+ - Added new Playlist UI to be default. ([#40911](https://github.com/brave/brave-browser/issues/40911))
```
Comment (kjozwiak) [inline CHANGELOG_iOS.md:13]: We usually use `Updated` when mentioning a feature that was overhauled re: new UI. Can probably just do something simple as per the following:

```
  - Updated Playlist UI. ([#40911](https://github.com/brave/brave-browser/issues/40911))ues/36801))
```

**Review by kjozwiak — CHANGES_REQUESTED** [review]
LGTM other than a few changes.

File: CHANGELOG_iOS.md (hunk 2 above)
Comment (Uni-verse) [reply CHANGELOG_iOS.md:15]: Good call!

**Comment by Uni-verse** [comment]
Thanks for the reviews @kjozwiak @srirambv. Changes are in. https://github.com/brave/brave-browser/commit/73e996421198417e3106e9ce4d117f66a619f8a3

**Review by kjozwiak — APPROVED** [review]
LGTM 👍 Nice work.
